components/navigation/BackButton.js

```diff
@@ -8,7 +8,7 @@ export default function BackButton({ text, className }) {
 
     return (
         <div className={ className }>
-            <span className={`back-button`} onClick={() => router.back()}>{ text }</span>
+            <span className={`back-button`} onClick={() => router.push('/')}>{ text }</span>

```

Comment:
This element probably has accessibility issues (I tried to check but couldn't find where this component is used)

Using an anchor element instead of a span element automatically gives screen readers relevant information (i.e. this element is a link and this element is labelled by its text content) so this change should automatically resolve accessibility issues:
```suggestion
            <a className={`back-button`} href="/">{ text }</a>
```

After I'm done with setting up automatic deployments, I'll set up accessibility ESLint rules

Reply:
Agreed! Let's merge this and fix accessibility issues separately.